errata_tool/build.py

```diff
@@ -22,33 +26,61 @@ def _fetch(self):
         """Fetch build data from Errata Tool API and store to properties"""
         self._data = self._get(self.url)
 
-        if self._data['released_errata'] is not None:
-            self._released_errata = Erratum(errata_id=self._data[
-                'released_errata']['id'])
+        if self._data:
+            if self._data['released_errata'] is not None:
+                self._released_errata_id = self._data['released_errata']['id']
+
+            if 'all_errata' in self._data:
+                self._all_errata_ids = [errata['id']
+                                        for errata in self._data['all_errata']]
+
+            self._signed_rpms = self._data.get('rpms_signed')
 
-        for errata_dict in self._data['all_errata']:
-            errata = Erratum(errata_id=errata_dict['id'])
-            self._all_errata.append(errata)
+            for et_file in self._data['files']:
+                self._files.append(et_file['path'])
 
-        self._signed_rpms = self._data.get('rpms_signed')
+            self._all_errata = None
+            self._released_errata = None
 
-        for et_file in self._data['files']:
-            self._files.append(et_file['path'])
+    @property
+    def all_errata_ids(self):
```

Comment:
Should probably add a test-case covering all_errata_ids() not retrieving advisories

Reply:
added one for coverage didn't do any verification that it didn't pull content though

Reply:
done